tests/Support/SupportCollectionTest.php

```diff
@@ -865,6 +865,21 @@ public function testTransform()
         $this->assertEquals(['first' => 'first-rolyat', 'last' => 'last-llewto'], $data->all());
     }
 
+    public function testTranspose()
+    {
+        $data = new Collection([
+            'name' => ['Tyrion Lannister', 'Daenerys Targaryen', 'Meera Reed'],
+            'house' => ['Lannister', 'Targaryen', 'Stark'],
+            'actor' => ['Peter Dinklage', 'Emilia Clarke', 'Ellie Kendrick'],
```

Comment:
I would remove the keys here because they just get stripped and aren't an important part of the operation. Maybe even add a separate explicit test like "testTransposeRemovesExistingKeys" that can serve as documentation?